charts/core/traefik/templates/_args.tpl

```diff
@@ -143,6 +143,13 @@ args:
   {{- end }}
   {{- if .Values.pilot.enabled }}
   - "--pilot.token={{ .Values.pilot.token }}"
+  {{- if .Values.pilot.enableThemePark -}}
+  {{- $pluginList := splitList "@" "github.com/packruler/traefik-themepark@v1.0.0" -}}
+  {{- $moduleName := first $pluginList -}}
+  {{- $version := last $pluginList }}
+  - "--experimental.plugins.themePark.modulename={{ $moduleName }}"
+  - "--experimental.plugins.themePark.version={{ $version }}"
+  {{- end }}
```

Comment:
This looks super messy with those variables.

Reply:
I agree 🤔 What do you think about this?
```
{{- $plugin := dict "module" "github.com/packruler/traefik-themepark" "version" "v1.0.0" -}}
- "--experimental.plugins.themePark.modulename={{ get $plugin "module" }}"
- "--experimental.plugins.themePark.version={{ get $plugin "version" }}"
```

Reply:
Just stop with the nonesenical dicts/list splitting.